tests/host_tools/metrics.py

```diff
@@ -46,52 +46,59 @@
 import json
 import os
 import socket
+from pathlib import Path
 from urllib.parse import urlparse
 
 from aws_embedded_metrics.constants import DEFAULT_NAMESPACE
 from aws_embedded_metrics.logger.metrics_logger_factory import create_metrics_logger
 
 
-class MetricsWrapperDummy:
-    """Send metrics to /dev/null"""
+class MetricsWrapper:
+    """A convenient metrics logger"""
+
+    def __init__(self, logger):
+        self.data = {}
```

Comment:
nit: could be `defaultdict(list)`, which'll save you some "if name not in self.data" statements

Reply:
I forgot to add the `unit` to the metrics, so now the logic of append is a bit more comlex